src/aleph/vm/storage.py

```diff
@@ -112,7 +112,7 @@ async def download_file(url: str, local_path: Path) -> None:
         ) as error:
             if attempt < (download_attempts - 1):
                 logger.warning(f"Download failed, retrying attempt {attempt + 1}/3...")
-                continue
+                # continue  #  continue inside try/finally block is unimplemented in `mypyc`
```

Comment:
Removing that line can break the behavior, it's needed to be tested.

Reply:
I checked the code, we are in a `for` loop and there is nothing executed after this line, so the `continue` was purely a hint for the reader anyways.